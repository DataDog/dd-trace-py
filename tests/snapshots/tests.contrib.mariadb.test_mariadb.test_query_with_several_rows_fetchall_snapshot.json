[[
  {
    "name": "mariadb.query",
    "service": "mariadb",
    "resource": "SELECT n FROM (SELECT 42 n UNION SELECT 421 UNION SELECT 4210) m",
    "trace_id": 0,
    "span_id": 1,
    "parent_id": 0,
    "type": "sql",
    "meta": {
      "db.name": "test",
      "db.user": "test",
      "out.host": "127.0.0.1",
      "runtime-id": "afb83c6e2adf41f9a723786fdb1b223b"
    },
    "metrics": {
      "_dd.agent_psr": 1.0,
      "_dd.measured": 1,
      "_dd.top_level": 1,
      "_dd.tracer_kr": 1.0,
      "_sampling_priority_v1": 1,
      "db.rowcount": 0,
      "out.port": 3306,
      "sql.rows": 0,
      "system.pid": 5857
    },
    "duration": 760000,
    "start": 1633039294966448000
  }],
[
  {
    "name": "mariadb.query.fetchall",
    "service": "mariadb",
    "resource": "SELECT n FROM (SELECT 42 n UNION SELECT 421 UNION SELECT 4210) m",
    "trace_id": 1,
    "span_id": 1,
    "parent_id": 0,
    "type": "sql",
    "meta": {
      "db.name": "test",
      "db.user": "test",
      "out.host": "127.0.0.1",
      "runtime-id": "afb83c6e2adf41f9a723786fdb1b223b"
    },
    "metrics": {
      "_dd.agent_psr": 1.0,
      "_dd.top_level": 1,
      "_dd.tracer_kr": 1.0,
      "_sampling_priority_v1": 1,
      "db.rowcount": 3,
      "out.port": 3306,
      "sql.rows": 3,
      "system.pid": 5857
    },
    "duration": 98000,
    "start": 1633039294967644000
  }]]
