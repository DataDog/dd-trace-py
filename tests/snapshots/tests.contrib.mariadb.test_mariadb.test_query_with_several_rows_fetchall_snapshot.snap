[[{"name" "mariadb.query"
   "service" "mariadb"
   "resource" "SELECT n FROM (SELECT 42 n UNION SELECT 421 UNION SELECT 4210) m"
   "type" "sql"
   "error" 0
   "span_id" 0
   "trace_id" 0
   "parent_id" nil
   "start" 1626718453355264000
   "duration" 1786000
   "meta" {"runtime-id" "0cbfb96da3ee4e86be3cf3b8a4004f86"
           "out.host" "127.0.0.1"
           "db.user" "test"
           "db.name" "test"}
   "metrics" {"_dd.agent_psr" 1.0
              "_dd.measured" 1
              "_sampling_priority_v1" 1
              "system.pid" 1502
              "out.port" 3306
              "db.rowcount" 0
              "sql.rows" 0
              "_dd.tracer_kr" 1.0}}]
 [{"name" "mariadb.query.fetchall"
   "service" "mariadb"
   "resource" "SELECT n FROM (SELECT 42 n UNION SELECT 421 UNION SELECT 4210) m"
   "type" "sql"
   "error" 0
   "span_id" 0
   "trace_id" 1
   "parent_id" nil
   "start" 1626718453359605000
   "duration" 113000
   "meta" {"runtime-id" "0cbfb96da3ee4e86be3cf3b8a4004f86"
           "out.host" "127.0.0.1"
           "db.user" "test"
           "db.name" "test"}
   "metrics" {"_dd.agent_psr" 1.0
              "_sampling_priority_v1" 1
              "system.pid" 1502
              "out.port" 3306
              "db.rowcount" 3
              "sql.rows" 3
              "_dd.tracer_kr" 1.0}}]]
